src/state.js

```diff
@@ -34,12 +35,16 @@ module.exports = class MonoplasmaState {
         this.members = initialMembers.map(m => new MonoplasmaMember(undefined, m.address, m.earnings))
         /** @property {MerkleTree} tree The MerkleTree for calculating the hashes */
         this.tree = new MerkleTree(this.members)
-
+        /** @property {string}  adminAddress the owner address who receives the admin fee and the default payee if no memebers */
+        this.adminAddress  = adminAddress
+        /** @property {number}  adminFeeFraction fraction of revenue that goes to admin */
+        this.setAdminFeeFraction(adminFeeFraction)
+        
         this.indexOf = {}
         this.members.forEach((m, i) => { this.indexOf[m.address] = i })
 
         // add default address in case revenue is received with no members
```

Comment:
I think every mention of "default" should be replaced with "admin"

Reply:
how about getDefaultPayee(){ return admin} ?

Reply:
done